### PR TITLE
Add Pulsar Community Weekly Update - 2020-10-17~ 2020-10-23

### PR DESCRIPTION
**Pull request by Huanli-Meng**

### CODE REVIEW

File: content/2020/2020-10/2020-10-23-pulsar-weekly.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+    https://github.com/apache/pulsar/pull/8067 ([@wolfstudy](https://github.com/wolfstudy))
+
+### Notable Bug Fix
+
+- [Managed Ledger] Fix the issue that the broker enters an infinite loop in `ManagedLedgerImpl.asyncReadEntries`.
+
+    https://github.com/apache/pulsar/pull/8284 ([@hangc0276](https://github.com/hangc0276))
+
+- [C++ Client] The client allocates more buffers than required.
```
Comment (Jennifer88huang) [inline content/2020/2020-10/2020-10-23-pulsar-weekly.md:87]: what's the result?

Comment (Huanli-Meng) [reply content/2020/2020-10/2020-10-23-pulsar-weekly.md]: updated, PTAL

File: content/2020/2020-10/2020-10-23-pulsar-weekly.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+id: "2020-10-23-pulsar-weekly"
+---
+
+## [Pulsar Community Weekly Update] 2020-10-17 ~ 2020-10-23
+
+This is the Pulsar community weekly update for 2020-10-17 ~ 2020-10-23, with updates on Pulsar client, broker, Pulsar Functions, transaction, tiered storage, and so on.
+
+### Pulsar Highlight
+
+Pulsar has supported acknowledging the batch message index since release 2.6.0. But Pulsar does not support setting the cursor. If users reset the cursor to a batch index of the batch message when consuming messages again, they will get the batch index again. Now, Pulsar supports resetting the cursor to a batch index of the batch message.
```
Comment (Jennifer88huang) [inline content/2020/2020-10/2020-10-23-pulsar-weekly.md:15]: Pulsar supports resetting the cursor to a batch index of the batch message now.
In 2.6.0 release, Pulsar supports acknowledging the batch message index, yet you cannot set the cursor. If you reset the cursor to a batch index of the batch message when consuming messages again, you will get the batch index again. Now you can reset the cursor now.

Comment (Jennifer88huang) [reply content/2020/2020-10/2020-10-23-pulsar-weekly.md:15]: 1. We can tell readers the highlights first.
2. Try to use present tense if possible.

**Review by Jennifer88huang — APPROVED** [review]

**Pull request merged**